dashboard/app/helpers/levels_helper.rb

```diff
@@ -1021,4 +1022,13 @@ def post_milestone_mode(post_milestone, post_failed_run_milestone)
       POST_MILESTONE_MODE.final_level_only
     end
   end
+
+  def get_documentation_environment
+    case @level.game
+    when Game.javalab
+      "javalab"
```

Comment:
nit: mind adding a comment that this should match the `name` of a ProgrammingEnvironment?

Reply:
yeah, I'm thinking it may be worth renaming this to `get_programming_environment` (and also renaming the variable to `programmingEnvironment`) to make things both clearer and more generic.

Reply:
oh I like that name better if you don't mind changing it. Documentation is a more catch-all term right now, so specifying programming environment would be nice

Reply:
done!